### PR TITLE
integrated with quovo ui widget to easily add new accounts

### PR DESCRIPTION
**Pull request by jdimatteo**
integrated with quovo ui widget to easily add new accounts

also renamed some things: transactions with connect, overview to snapshot, and profile to settings

------

I already deployed it; please try it out as part of this PR review

image

### CODE REVIEW

File: web_app/api/quovo.py [hunk 1]
```diff
@@ -15,21 +15,30 @@ def quovo(get_or_post_path=None, json=None, delete_path=None, put_path=None):
         raise Exception("Invalid argument combination")
     requests_func = requests.delete if delete_path else (
         requests.put if put_path else (
-            requests.post if json else requests.get
+            requests.post if json is not None else requests.get
```
Comment (anfedorov) [inline web_app/api/quovo.py]: could do a retrospective on how this made it into prod

Comment (jdimatteo) [reply web_app/api/quovo.py]: fyi I find this comment is frustratingly premature given that we only have 15 commits (and all but this merge commit is from me)

Comment (anfedorov) [reply web_app/api/quovo.py]: sure. at what point should it become no longer OK to commit buggy code into production?

**Comment by jdimatteo** [comment]
As soon as you start committing code

**Comment by jdimatteo** [comment]
Lol, I didn’t mean it to be round about, please I desparately want you to
commit some code!

**Pull request merged**